packages/typespec-ts/test/integration/generated/projection/src/clientDefinitions.ts

```diff
@@ -2,42 +2,68 @@
 // Licensed under the MIT license.
 
 import {
-  JsonProjectionParameters,
-  ClientProjectionParameters,
-  LanguageProjectionParameters,
+  PropertyJsonParameters,
+  PropertyClientParameters,
+  PropertyLanguageParameters,
+  PropertyJsonAndClientParameters,
+  OperationParameters,
+  ParameterParameters,
 } from "./parameters";
 import {
-  JsonProjection204Response,
-  ClientProjection204Response,
-  LanguageProjection204Response,
+  PropertyJson204Response,
+  PropertyClient204Response,
+  PropertyLanguage204Response,
+  PropertyJsonAndClient204Response,
+  Operation204Response,
+  Parameter204Response,
 } from "./responses";
 import { Client, StreamableMethod } from "@azure-rest/core-client";
 
-export interface JsonProjection {
+export interface Json {
   post(
-    options?: JsonProjectionParameters
-  ): StreamableMethod<JsonProjection204Response>;
+    options?: PropertyJsonParameters
+  ): StreamableMethod<PropertyJson204Response>;
 }
 
-export interface ClientProjection {
+export interface _client {
```

Comment:
Not sure this is a proper way to handle this interface name

Reply:
I think it's fine, because this interface will not be actually used by customer in normal case.